packages/sw-background-sync-queue/src/lib/responseManager.js

```diff
@@ -0,0 +1,17 @@
+import idbQHelper from './idbHelper';
+
+class ResponseManager {
+	async putResponse(hash, idbObject, response) {
+		idbObject.response = {
+			headers: JSON.stringify(response.headers),
+			status: response.status,
+			body: await response.blob(),
+		};
+		idbQHelper.put(hash, idbObject);
+	}
+	getResponse(reqId) {
+	}
+}
+
+const resManager = new ResponseManager();
+export default resManager;
```

Comment:
I don't see any reason to wrap this in a `ResponseManager` class. If you have a helper function that doesn't need to keep track of state information, just put it inside a standalone module.